src/internal/measures_utils.cc

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+#include "measures_utils.h"
+
+#include <opencensus/exporters/stats/prometheus/prometheus_exporter.h>
+#include <prometheus/text_serializer.h>
+
+namespace google {
+namespace internal {
+
+namespace {
+
+::prometheus::TextSerializer text_serializer;
```

Comment:
Are text_serializer and exporter trivially constructible? Maybe make them both static pointers inside `SerializeMetricsToPrometheusTextFormat`?

Reply:
now creating objects every time when `SerializeMetricsToPrometheusTextFormat` is called, because I cannot make sure whether my implementation on static pointer is the right approach, so eventually switch to object approach.

As for now, `SerializeMetricsToPrometheusTextFormat()` will only be called when server is called on `/metrics` specifically, so I don't think it will have huge performance impact and don't want to optimize early without my solid knowledge on static pointer approach.